splitio/storage/redis.py

```diff
@@ -180,27 +180,22 @@ def get_split_names(self):
             _LOGGER.debug('Error: ', exc_info=True)
             return []
 
+    def get_splits_count(self):
+        """
+        Return splits count.
+
+        :rtype: int
+        """
+        return 0
+
     def get_all_splits(self):
         """
         Return all the splits in cache.
 
-        :return: List of all splits in cache.
-        :rtype: list(splitio.models.splits.Split)
+        :return: 0
```

Comment:
we are using this method for manager, why you removed the logic?

Reply:
I added it back